app/src/androidTest/java/mozilla/lockbox/ItemDetailsTest.kt

```diff
@@ -0,0 +1,40 @@
+package mozilla.lockbox
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import mozilla.lockbox.robots.itemDetail
+import mozilla.lockbox.robots.itemList
+import mozilla.lockbox.view.RootActivity
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+open class ItemDetailsTest {
+    private val navigator = Navigator()
+
+    @Rule
+    @JvmField
+    val activityRule: ActivityTestRule<RootActivity> = ActivityTestRule(RootActivity::class.java)
+
+    @Test
+    fun itemDetailsAreDisplayed() {
+        navigator.gotoItemList(false)
+        itemList { selectItem() }
+        itemDetail { exists() }
+    }
+
+    @Test
+    fun showToastWhenUsernameCopied() {
+        navigator.gotoItemDetail()
+        itemDetail { tapCopyUsername() }
+        itemDetail { toastIsDisplayed(R.string.toast_username_copied, activityRule) }
+    }
+
+    @Test
+    fun showToastWhenPassCopied() {
+        navigator.gotoItemDetail()
+        itemDetail { tapCopyPass() }
+        itemDetail { toastIsDisplayed(R.string.toast_password_copied, activityRule) }
+    }
+}
```

Comment:
Nice. Is there a reason why you're doing this in this PR?

Reply:
codecov complained and I think the unit test that I added was enough, but I already started to add some UI tests so I pushed them also. I can add them in another PR if you think it is better

Reply:
This is fine. :)